src/core/RecyclerListView.js

```diff
@@ -498,5 +500,8 @@ RecyclerListView.propTypes = {
     disableRecycling: PropTypes.bool,
 
     //Default is false, if enabled dimensions provided in layout provider will not be strictly enforced. Rendered dimensions will be used to relayout items. Slower if enabled.
-    forceNonDeterministicRendering: PropTypes.bool
+    forceNonDeterministicRendering: PropTypes.bool,
+
+    //Web only. Invert scrollable view. Default  is false
```

Comment:
Can easily be done for react-native as well along similar lines

Reply:
Probably yes, I just didn't have the time to setup a RN test case. Maybe I can try to do it once web is working.

Reply:
Yeah sure, we should also be done with TypeScript by then